examples/RGBWW: needs @filter to skip tiny/low memory boards (overflows attiny85 by 46KB)

## Symptom

Every AVR/small-MCU CI build on `master` fails compiling the new `examples/RGBWW/RGBWW.ino` (added in #2560, commit `9ae94ae14`):

```
ld: region `text' overflowed by 46396 bytes
collect2: error: ld returned 1 exit status
FAILED: RGBWW
ERROR: Compilation failed for board attiny85
```

Reproducible run: https://github.com/FastLED/FastLED/actions/runs/26677701977 (and all other AVR workflows on the same sweep).

## Root cause

`examples/RGBWW/RGBWW.ino` ships with **no `@filter` block**, so the CI test matrix tries to compile it on every board, including tiny-tier AVRs that can't possibly hold it.

Total code size needed: ~54 KB (46 KB overflow + 8 KB flash). Won't fit on:

| Board       | Tier | Flash |
|-------------|------|-------|
| attiny85    | tiny | 8 KB  |
| attiny88    | tiny | 8 KB  |
| attiny4313  | tiny | 4 KB  |
| attiny1604  | tiny | 16 KB |
| atmega8a    | tiny | 8 KB  |
| uno / nano  | low  | 32 KB |
| uno_r4_*    | low  | varies |

The example pulls in `fl/channels/channel.h`, `fl/channels/config.h`, `fl/gfx/rgbww.h` and the colorimetric RGB→RGBW solvers — heavy templated code that the linker can't dead-strip enough of.

## Affected jobs

Every AVR / small-MCU board workflow on master: `attiny85`, `attiny88`, `attiny1604`, `attiny1616`, `atmega8a`, plus the size guards `check_uno_size`, `attiny85_binary_size`, etc.

## Proposed fix

One-line `@filter` at the top of `examples/RGBWW/RGBWW.ino`:

```cpp
// @filter: (memory is large)
```

`(memory is large)` matches boards in the `large` or `huge` tier (Apollo3 / nRF52 / SAMD21 / SAMD51 / STM32F4+ / teensy 3.5+ / teensy 4.x / ESP32 / native / web / rp2040). It excludes `tiny` and `low`, matching the existing pattern used by `examples/Animartrix/Animartrix.ino` and `examples/AnimartrixRing/AnimartrixRing.ino` for similarly heavy examples.

## Repro

`bash compile attiny85 --examples RGBWW`

---
Filed automatically while triaging broken builds; intent is to fix and open a PR immediately.

--- a/examples/RGBWW/RGBWW.ino
+++ b/examples/RGBWW/RGBWW.ino
@@ -1,6 +1,14 @@
 // FL_AGENT_ALLOW_NEW_EXAMPLE — first example sketch for the 5-channel
 // RGB + warm-W + cool-W path landed in PR #2560 (issue #2558).
 //
+// @filter: (memory is large)
+//
+// Pulls in the full Channels API plus the colorimetric RGB→RGBWW solvers;
+// total code size is ~54 KB on AVR (overflows attiny85's 8 KB and uno's
+// 32 KB flash). Restrict CI compile matrix to boards with >= "large"
+// memory tier (Apollo3 / nRF52 / SAMD21+ / SAMD51 / STM32F4+ /
+// teensy 3.5+ / teensy 4.x / ESP32 / native / web / rp2040).
+//
 // Add the following to platformio.ini for full colorimetric output:
 //     build_flags =
 //         -DFASTLED_RGBW_COLORIMETRIC=1     ; provides the underlying math

